### ISSUE TITLE
account name leaked on lock screen

### ISSUE THREAD

**Issue by QuelonaSec** [siacs/Conversations#2795]
#### General information

Version: 1.23.6

#### Steps to reproduce

1. force account to use Tor
2. shut down orbot
3. lock device
4. wait a while
5. check lock screen

#### Expected result

especially if someone is using Tor, you'd expect your privacy to be respected.
your xmpp account shouldn't be revealed to 3rd partys (not having the pin/password of a locked device).

#### Actual result

there's a lock screen message saying, that there's trouble connecting (that might be okay, i think) and additionally it prints the account name (e.g. veryprivateaccount@conversations.im).

**Comment by licaon-kter** [comment]
Use your Android notifications settings to either fully hide or hide the content of the notifications.


/close as invalid

**Comment by QuelonaSec** [comment]
I disagree. Actually I selected "hide sensitive notification content" with the result of leaking the (imo sensitive) Account name.

**Comment by licaon-kter** [comment]
What device? What Android version?

I don't see this in Xperia 5.1.1 or 6.0.1.

**Comment by QuelonaSec** [comment]
Nexus 7, Android 7.1.2 (fully patched Lineage OS 14.1).
Do you need more information?
